UnityProject/Assets/Scripts/Items/Implants/Organs/Lungs.cs

```diff
@@ -183,7 +183,7 @@ private bool BreatheIn(GasMix breathGasMix, ReagentMix blood, float efficiency)
 			RelatedPart.HealthMaster.HealthStateController.SetSuffocating(true);
 			if (Random.value < 0.2)
 			{
-				Chat.AddActionMsgToChat(gameObject, "You gasp for breath", $"{RelatedPart.HealthMaster.gameObject.ExpensiveName()} gasps");
+				Chat.AddActionMsgToChat(RelatedPart.HealthMaster.gameObject, "You gasp for breath", $"{RelatedPart.HealthMaster.gameObject.ExpensiveName()} gasps");
```

Comment:
what is this change?

Reply:
The player would always see "You gasp for breath", $"{RelatedPart.HealthMaster.gameObject.ExpensiveName()} gasps"instead of "You gasp for breath" as the lungs object isnt the right performer player gameobject